kalite/distributed/static/js/distributed/color_palette/color_palette.js

```diff
@@ -0,0 +1,127 @@
+var $ = require("../base/jQuery");
+require("browsernizr/test/inputtypes");
+var Modernizr = require("browsernizr");
+var colorPaletteTemplate = require("./hbtemplates/color-palette.handlebars");
+var BaseView = require("../base/baseview");
+
+var ColorPaletteView = BaseView.extend({
+    template: colorPaletteTemplate,
+    tagName: "style", 
+
+    initialize: function() {
+        this.render();
+    },
+
+    render: function(){
+        this.$el.html(this.template({
+            k_bg_clr: $("#my_background").val(),
```

Comment:
Where are these colours getting picked from? Would be good to pull a lot of the code out from below and include it in this view, so that we can use Backbone events to handle the value changes etc.